### PR TITLE
Fix project loading with bad text-number connections

### LINKED ISSUE #1213: Validation problems prevent project loading
Two problems in the validation of the blocks editor that can result in invalid workspaces on load:

Normally, a text block with numbers will be accepted as numeric thanks to type coercion. This is done by using `parseFloat`, which will parse as many valid characters as possible. This doesn't guarantee that the value is a valid number, however, for example:

image

In addition to doing this, once the block is connected you can remove the number and the addition block accepts the string:

image

When loading a project with blocks like this, the error "The blocks area did not load properly..." error appears and the project is no longer editable.

### PR DESCRIPTION
**Pull request by ewpatton**
Fixes #1213 

Change-Id: I78c1666fd34346e46e68e112d02603e4de197fa5

### CODE REVIEW

**Comment by ewpatton** [comment]
This bug has come up a number of times on the forum. Here's the most recent example: https://community.appinventor.mit.edu/t/the-blocks-area-did-not-load-properly-changes-to-the-blocks-for-screen-5690035441500160-scrhoy-will-not-be-saved/7729?u=ewpatton

**Comment by BeksOmega** [comment]
I like the tests :D

File: appinventor/blocklyeditor/src/workspace_svg.js [hunk 1]
```diff
@@ -471,13 +471,15 @@ Blockly.WorkspaceSvg.prototype.loadBlocksFile = function(formJson, blocksContent
   if (blocksContent.length != 0) {
     try {
       Blockly.Events.disable();
+      this.isLoading = true;
```
Comment (ellelili2025) [inline appinventor/blocklyeditor/src/workspace_svg.js:474]: Can you help to explain why this is needed?

Comment (ellelili2025) [reply appinventor/blocklyeditor/src/workspace_svg.js:474]: Never mind. I got it

**Pull request merged**